pkg/auth/jwt.go

```diff
@@ -0,0 +1,47 @@
+package auth
+
+import (
+	"bytes"
+	"context"
+	"log"
+	"io/ioutil"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/grpc-ecosystem/go-grpc-middleware/auth"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var (
```

Comment:
For stuff like bb-storage I try to structure everything as a bag of classes as much as possible. Would there be a way for us to eliminate these global variables?

Reply:
Should be better now: I've moved this to a `struct` implementing an interface that will also serve as token validation cache (WIP).